backend/tracim_backend/command/cleanup.py

```diff
@@ -247,7 +247,7 @@ def _delete_user_database_info(
         anonymize_if_required: bool = False,
         anonymized_user_display_name: typing.Optional[str] = None,
     ):
-        print('trying to delete user {}: "{}"\n'.format(user.user_id, user.email))
+        print('trying to delete user {}: "{}"\n'.format(user.user_id, user.email or user.username))
```

Comment:
and here we decide to show email first ?

Reply:
see https://github.com/tracim/tracim/pull/2957/commits/cfb7110df168a2d6b2a8ca3675cb4c481c911ebd

Reply:
ok